menu.json

```diff
@@ -1,249 +1,232 @@
+          "value": "{{installed_version}}",
+          "indent": 145,
+          "color": "black"
         },
         {
-            "date": "{{release_date}}",
-            "name": "Updates for Open Rails",
-            "description": "No special checks for the latest version",
-            "updateChannel": "unstable",
-            "prefixItemList": [
-                {
-                    "$type": "ORTS.Record, Menu",
-                    "label": "New version available",
-                    "value": "{{new_version}}",
-                    "indent": 140,
-                    "color": "black"
-                },
-                {
-                    "$type": "ORTS.Record, Menu",
-                    "label": "Installed version",
-                    "value": "{{installed_version}}",
-                    "indent": 140,
-                    "color": "black"
-                },
-                {
-                    "$type": "ORTS.Record, Menu",
-                    "label": "Update mode",
-                    "value": "{{update_mode}}   (To change Update Mode, use Options > System.)",
-                    "indent": 140,
-                    "color": "black"
-                }
-            ],
-            "metLists": {
-                "itemList": [
-                    {
-                        "$type": "ORTS.Link, Menu",
-                        "label": "What's new",
-                        "value": "Find out on-line what's new in this version.",
-                        "url": "https://james-ross.co.uk/projects/or",
-                        "indent": 140,
-                        "color": "black"
-                    },
-                    {
-                        "$type": "ORTS.Update, Menu",
-                        "label": "Install",
-                        "value": "Install the new Unstable version",
-                        "indent": 140,
-                        "color": "black"
-                    }
-                ],
-                "checkIdList": [
-                    {
-                        "id": "not_updated"
-                    }
-                ]
-            },
-            "suffixItemList": [
-                {
-                    "$type": "ORTS.Text, Menu",
-                    "label": "(To change Update Mode, use Options > System.)",
-                    "indent": 140,
-                    "color": "black"
-                }
-            ]
+		      "$type":"ORTS.Record, Menu",
+          "label": "Update mode",
+          "value": "{{update_mode}}   (To change Update Mode, use Options > System.)",
+          "indent": 145,
+          "color": "black"
         }
-    ],
-    "checkList": [
+      ],
+      "metLists": {
+        "itemList": [
+          {
+            "$type":"ORTS.Link, Menu",
+            "label": "What's new",
+            "value": "Find out on-line what's new in this version.",
+            "Url": "https://www.openrails.org/discover/latest_stable_version",
```

Comment:
Did this key want to be `url` (all lower-case) as well?

Reply:
Eagle-eyes !

Corrected. Thanks, James.